crates/orderbook/src/main.rs

```diff
@@ -509,6 +517,13 @@ async fn main() {
                         }),
                         &estimator.name(),
                     )),
+                    PriceEstimatorType::OneInch => Box::new(instrumented_and_cached(
+                            Box::new(OneInchPriceEstimator::new(
+                                Arc::new(OneInchClientImpl::new(OneInchClientImpl::DEFAULT_URL, client.clone()).unwrap()),
+                                args.disabled_one_inch_protocols.clone()
+                            )),
+                            &estimator.name(),
```

Comment:
Indentation feels off. Surprised `cargo fmt` didn't catch this.

Reply:
When I run cargo fmt locally it doesn't work properly on the big main files. I guess CI has the same problem.

Reply:
`rustfmt` doesn't give me any error message for that particular file. I just fixed the indentation manually now.